Configuration/TCA/Overrides/pages.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+defined('TYPO3') or die();
+
+call_user_func(function () {
+    $GLOBALS['TCA']['pages']['columns']['backend_layout']['exclude'] = 0;
+});
```

Comment:
i think we should give a hint in the README, and not change Access-Right from Core.